static/app/views/alerts/rules/metric/details/metricChart.tsx

```diff
@@ -536,6 +538,27 @@ class MetricChart extends PureComponent<Props, State> {
       moment.utc(timePeriod.end).add(timeWindow, 'minutes')
     );
 
+    let activationFilter = '';
+    if (
+      rule.monitorType === MonitorType.ACTIVATED &&
+      selectedIncident &&
+      selectedIncident.activation
+    ) {
+      const activation = selectedIncident.activation;
+      const activator = activation.activator;
+      const conditionType = activation.conditionType;
```

Comment:
Nit: Recommend using destruct for this, it's a bit simpler as you can DRY the variable names, and then remove the 2nd line creating conditionType (and then easier in the future if there's more to destruct)
```suggestion
      const { activation } = selectedIncident;
      const { activator, conditionType } = activation;
```

Reply:
ohya duh 👍